docker-compose.yaml

```
@@ -38,9 +38,6 @@ services:
       - backend
     volumes:
       - ./musclemate-frontend/src/:/app/src
```

```suggestion
      - ./musclemate-frontend/:/app/
```

Quita esto del tiron y asi tienes un solo volumen